README.md

```diff
@@ -40,10 +40,151 @@ Note: If you use the Drush command, it is advisable to Move (not copy) the insta
 
 ## Configuration
 
+### Djatoka Image Server
 Set the paths for 'Djatoka server base URL' and configure OpenSeadradon in Administration » Islandora » OpenSeadragon (admin/islandora/module).
 
 ![Configuration](https://camo.githubusercontent.com/c1bf991b5cc758a4420444564a91b286007e6f6e/687474703a2f2f692e696d6775722e636f6d2f4e6566597169432e706e67)
 
+If you have an *existing* install it's required to update Openseadragon to it's latest version. You can do this quickly 
+with the provided Drush command.
+
+```bash
+drush openseadragonplugin
+```
+
+### Loris Image Server
+
+The [Loris Image server](https://github.com/loris-imageserver/loris) currently only works when configured to directly 
+communicate with Fedora. As such it does not work with Islandora's authentication system, meaning it can only be used to 
+display public datastreams, or it can be configured to give full access to show anything in the system. This does not 
+give any user the ability to access any datastream however, just the JPEG generated for display in the browser, not the 
+original.
+
+Additionally there are issues with it's caching system in that it needs to be manually flushed please read the project 
+[documentation](https://github.com/loris-imageserver/loris/blob/development/doc/cache_maintenance.md) for more 
+information.
+
+Provided below is instructions for installing loris on Ubuntu 14.04, the commands / packages may change slightly 
+depending on your setup.
+
+#### Create loris user
```

Comment:
I'm interested in the work you've been doing here: https://github.com/jpstroop/loris-redux, in particular what you have under the goal section. I had a go with it but didn't successfully get it going (that being said I hadn't put in a huge amount of time sorting out the dependancies and what have you). I'm keen on using Varnish and having authentication support. Seems a bit easier to write a plugin for Islandora with the layout of loris-redux repository.